nominal/_utils.py

```diff
@@ -7,10 +7,12 @@
 from dataclasses import dataclass
 from datetime import datetime, timezone
 from pathlib import Path
-from typing import BinaryIO, Iterable, Iterator, Literal, NamedTuple, TypeVar, Union
+from typing import BinaryIO, Iterable, Iterator, Literal, NamedTuple, Type, TypeVar, Union
 
 import dateutil.parser
-from typing_extensions import TypeAlias  # typing.TypeAlias in 3.10+
+from typing_extensions import TypeAlias
+
+from nominal._api.combined import scout_video_api  # typing.TypeAlias in 3.10+
```

Comment:
```suggestion
from typing_extensions import TypeAlias # typing.TypeAlias in 3.10+
```

not quite sure how the comment moved...but also this second import is unused